po/wesnoth-wof/fr.po

```diff
@@ -3583,7 +3799,7 @@ msgstr ""
 "Resha, peut-être que ... toutes ces patrouilles t'ont épuisée.\n"
 "\n"
 "Je te libère de tes obligations.\n"
-"Retourne dans ton domaine pour y retrouver tes sens."
+"Retourne dans ton aire pour y retrouver tes sens."
```

Comment:
Original dialogue :
"Perhaps... your patrols have been overtaxing, Resha.\n"
"\n"
"I relieve you of all duties.\n"
"Return to the eyrie until your sense recovers."

tradution plus proche: 
« Peut-être... que tes patrouilles t'ont trop épuisée, Resha. »
« »
« Je te décharge de toutes tes fonctions. »
« Retourne au refuge jusqu'à ce que tu aies repris tes esprits. »